guides/source/installing_ruby.md

```diff
@@ -0,0 +1,105 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
+
+Installing Ruby Guide
+=====================
+
+This guide will walk you through installing the Ruby programming language on your operating system.
+
+While your OS might come with Ruby pre-installed, it's often outdated and can't be upgraded. Using a version manager like [Mise](https://mise.jdx.dev/getting-started.html) allows you to install the latest Ruby version, use a different Ruby version for each app, and easily upgrade to new versions when they're released.
+
+--------------------------------------------------------------------------------
+
+## Choose your operating system
+
+Follow the section for the operating system you use:
+
+* [macOS](#installing-ruby-on-macos)
+* [Ubuntu](#installing-ruby-on-ubuntu)
+* [Windows](#installing-ruby-on-windows)
+
+NOTE: Any commands prefaced with a dollar sign `$` should be run in the command line.
+
+### Installing Ruby on macOS
+
+For macOS, you'll need Xcode Command Line Tools and Homebrew to install dependencies needed to compile Ruby.
+
+Open Terminal and run the following commands:
+
+```shell
+# Install Xcode Command Line Tools
+$ xcode-select --install
+
+# Install Homebrew and dependencies
+$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+$ echo 'export PATH="/opt/homebrew/bin:$PATH"' >> ~/.zshrc
+$ source ~/.zshrc
+$ brew install openssl@3 libyaml gmp rust
+
+# Install Mise version manager
+$ curl https://mise.run | sh
+$ echo 'eval "$(~/.local/bin/mise activate)"' >> ~/.zshrc
+$ source ~/.zshrc
+
+# Install Ruby globally with Mise
+$ mise use -g ruby@3
+```
+
+### Installing Ruby on Ubuntu
+
+For Ubuntu, open Terminal and run the following commands:
```

Comment:
Ubuntu Jammy (22.04) libssl is 3.0 so we should recommend a minimum version of Ubuntu or the latest LTS release.